app/controllers/legal_docs_controller.rb

```diff
@@ -24,10 +24,8 @@ def create
 
   def index
     smart_listing_create :legal_docs, 
-                        LegalDoc.accessible_by(current_ability), 
-                        partial: "legal_docs/listing", 
-                        default_sort: { created_at: "desc" }
-    
+                        LegalDoc.all, 
```

Comment:
undefined local variable or method `smart_listing' - referencing the views/legal_docs/_listing.html.haml file

I basically mimic the Employees controller.